src/python_routing_v02_diffusive/nwis_client/__init__.py

```diff
@@ -0,0 +1 @@
+from .iv import IVDataService
```

Comment:
Better yet, just install it from here:
https://pypi.org/project/hydrotools.nwis-client/